governance/votes/Executive vote - February 07, 2024.md

```diff
@@ -0,0 +1,125 @@
+- **100,603 DAI** will be paid to Aave as part of the SparkLend revenue share.
+- The HV Bank (RWA009-A) `doc` parameter will be updated.
+- Ten Aligned Delegates will receive compensation for January 2024.
+
+**Voting for this executive proposal will place your MKR in support of the changes and additions outlined above.**
+
+Unless otherwise noted, the changes and additions listed above are subject to the [GSM Pause Delay](https://manual.makerdao.com/parameter-index/core/param-gsm-pause-delay). This means that if this executive proposal passes, the changes and additions listed above will only become active in the Maker Protocol after the GSM Pause Delay has expired. The GSM Pause Delay is currently set to **48 hours**.
+
+If this executive proposal does not pass within 30 days, then it will expire and can no longer have any effect on the Maker Protocol.
+
+
+---
+
+## Proposal Details
+
+### Auction Parameter Updates
+
+* **Authorization**: [Governance Poll](https://vote.makerdao.com/polling/QmWLyYW7)
+* **Proposal**: [Forum post "[Stability Scope Auction Parameters Changes #1] Liquidation Throughput Limit"](https://forum.makerdao.com/t/stability-scope-auction-parameters-changes-1-liquidation-throughput-limit/23508)
+
+If this executive proposal passes, the following auction parameter changes will be changed:
+
+#### [Local Liquidation Limit (ilk.hole)](https://mips.makerdao.com/mips/details/MIP104#14-3-1-5-8-local-liquidation-limit-hole-) Changes
+
+* Increase the WSTETH-A Local Liquidation Limit (ilk.hole) by 15 million DAI from 15 million DAI to **30 million DAI**.
+* Increase the WSTETH-B Local Liquidation Limit (ilk.hole) by 10 million DAI from 10 million DAI to **20 million DAI**.
+* Decrease the WBTC-A Local Liquidation Limit (ilk.hole) by 20 million DAI from 30 million DAI to **10 million DAI**.
+* Decrease the WBTC-B Local Liquidation Limit (ilk.hole) by 5 million DAI from 10 million DAI to **5 million DAI**.
+* Decrease the WBTC-C Local Liquidation Limit (ilk.hole) by 10 million DAI from 20 million DAI to **10 million DAI**.
+
+#### [Global Liquidation Limit (Hole)](https://manual.makerdao.com/parameter-index/core/param-global-liquidation-limit) Change
+
+* Increase the Global Liquidation Limit (Hole) by 50 million DAI from 100 million DAI to **150 million DAI**.
+
+### GUSD Input Conduit Management
+
+* **Authorization**: [Ecosystem Appoval](https://forum.makerdao.com/t/executive-spell-gusd-input-conduit-management/23597/2)
+* **Proposal**: [Forum post "Executive Spell GUSD Input Conduit Management"](https://forum.makerdao.com/t/executive-spell-gusd-input-conduit-management/23597)
+
+If this executive proposal passes, the following actions will be taken to `push` the remaining GUSD out of the PSM_GUSD_INPUT_CONDUIT_JAR, exchange it for Dai, and add it to the Surplus Buffer:
+
+1. Raise Debt Ceiling of PSM_GUSD_A to 567,660 DAI - accounts for the outstanding debt of 0.91 DAI of the vault.
```

Comment:
The wording here is a bit awkward. Perhaps a clearer way to express this would be, "accounting for the GUSD rewards and the remaining 0.91 DAI still present in the vault."